examples/federated-learning/tff/nvflare/README.md

```diff
@@ -0,0 +1,205 @@
+# Nvflare with TensorFlow
```

Comment:
Reminder to update the PR title and description to summarize the readme.

Reply:
Done